### ISSUE TITLE
Black and white should be color palette options

### ISSUE THREAD

**Issue by foxharp** [flameshot-org/flameshot#2553]
### Feature Description

I know the new picker allows using any color, but black and white are pretty basic selections for marking up either dark or bright images.

**Comment by mmahmoudian** [comment]
Duplicate of:

- #721
- #1215
- #1942

You can add the colors you want in the config so that you always have it in the color wheel.

**Comment by foxharp** [comment]
Well, I don't see how to affect the color wheel in the configuration options, and I have no idea where flameshot stores its config (especially when using the snap version), but I did find the "Use large predefined color palette" checkbox, which solves my problem, except that I can't seem to save the new configuration, again probably because it's a snap.

**Comment by foxharp** [comment]
Weird.  The configuration seems to have worked (I'm getting the large palette), but when I enter the configuration tool again, the checkbox is no longer checked.

**Comment by mmahmoudian** [comment]
> Well, I don't see how to affect the color wheel in the configuration options

Which version are you using 

> I have no idea where flameshot stores its config (especially when using the snap version)

The config file is always stored at `~/.config/flameshot/flameshot.ini` regardless of the method of installation 

> did find the "Use large predefined color palette" checkbox, which solves my problem

That would show a very large set of colors. In the newer version the colors are configurable in graphical configuration window. I don't remember from the top of my head if it is in v11.0.0 or not. I think it is not, but you still can get it either by compiling the Flameshot or by downloading the nightly builds and install them:

https://flameshot.org/docs/installation/development-build/

We are discussing to have a new version released soon.

> The configuration seems to have worked (I'm getting the large palette), but when I enter the configuration tool again, the checkbox is no longer checked

This is not an internet behaviour and is potentially a bug. It would be nice if you can fill a big report for it so we can investigate it further. Thank you.

**Comment by foxharp** [comment]
I think my problems may have been user error, resulting from running flameshot as root -- I didn't have a flameshot.ini in my .config/ , but there was one under /root.  I purged the snap.  While looking for another copy to download, I discovered that you also have .deb files for current builds.  I'd much rather deal with apt than snap, so that's what I installed this time, and all is well.  Sorry for the noise.

**Comment by mmahmoudian** [comment]
Typically (unless the user is Arch Linux or Arch-derived Linux user) we don't know what version they are using and tbh I don't remember what was the available features in version 0.10.0 for example, so it is very hard to tell user what to do when the version is much older than what we are working on (unless of course I go and install that old version on a VM, which is too much work for so little gain imho.

Anyways, I'm happy that the issue is resolved. Now that I'm behind a computer, I can share you this part of my personal config with which you can customize the colors in the color wheel:

```ini
[General]
userColors=picker, #800000, #ff0000, #ffff00, #00ff00, #008000, #00ffff, #0000ff, #ff00ff, #800080, #fa0164
```
